Kajabi/sage-lib
# feat(colors): update to new color specs

## Issue
Closes #938 

## Description

Update colors per design specs.
 https://www.figma.com/file/9Km09NjlZHYWsMP7EGT8tI/Sage-3-for-Admin?node-id=3554%3A25

## Screenshots
### ✨ Color updates ✨
|  Before  |  After  |
|--------|--------|
| ![before-colors](https://user-images.githubusercontent.com/24237393/138935952-f6481a09-cef8-4d58-bb7c-dbfd617ed633.png) | ![after-colors](https://user-images.githubusercontent.com/24237393/138935975-960da658-e62e-463c-b2f3-088c59a2346a.png) |


## Testing in `sage-lib`
1. View colors at http://localhost:4000/pages/foundations/color
2. Confirm new colors match design specs https://www.figma.com/file/9Km09NjlZHYWsMP7EGT8tI/Sage-3-for-Admin?node-id=3554%3A25

## Testing in `kajabi-products`
(**LOW**) kajabi/sage-lib#938 - no testing in `kajabi-products`; Sage color updates only.
